variables/_z-index.pcss

```diff
@@ -11,5 +11,6 @@
 	--z-index-scroller: 7;
 	--z-index-week-event-hover: 5;
 	--z-index-map-event-hover: 5;
+	--z-index-map-event-hover-actions: 6;
```

Comment:
I wonder if we should spread the other z-indices out so we have spaces between 5, 6, and 7. If we have any z-index needed between any of these numbers, we'll need to move things around.

Reply:
I mean maybe we should do it now rather than later.

Reply:
I though that at some point, but given that are related to other views I thought that by only making it higher that `event-hover` is more than enough. (And also smaller than the index for the scroller).

Reply:
Sure, fair enough. Just something for us to think about next time we add z-indices in between other z-indices.

Reply:
Also once we share this with Green Team we're going to have to really watch how they work with each other.